src/shared/forms/address-validators.ts

```diff
@@ -6,7 +6,7 @@ import { isEmptyString, isUndefined } from '@leather.io/utils';
 
 import { FormErrorMessages } from '@shared/error-messages';
 
-export function nonEmptyStringValidator(message = FormErrorMessages.AddressRequired) {
```

Comment:
Yeah looks wrong to me that this validator would have that default value

Reply:
why wrong? this is just validation message

Reply:
yeah, empty string validator shouldn't have that default value. don't recall why put it there